models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_kyberswap_pools.sql

```diff
@@ -0,0 +1,278 @@
+{{ config(
+    schema='lido_liquidity_arbitrum',
+    alias = 'kyberswap_pools',
+    partition_by = ['time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'time'],
```

Comment:
By the way, why is `time` part of `unique_key` here?
I am thinking `pool` pool address alone may be enough for `unique_key`?

Reply:
we need time in key because for each pool we have daily metrics:
<pool1> <time1> <set of metrics>
<pool1> <time2> <set of metrics> 
<pool2> <time1> <set of metrics>

Reply:
I see.